tests/lib/prepare.sh

```diff
@@ -3,7 +3,8 @@
 set -eux
 
 prepare_classic() {
-    apt install -y ${SPREAD_PATH}/../snapd_*.deb
+    # relying on dpkg as apt(-get) does not support installation from local files in trusty.
+    dpkg -i ${SPREAD_PATH}/../snapd_*.deb || apt-get -f install -y
```

Comment:
As commented on another PR, let's please not do this as it's guessing the reason for the failure and introducing an alternative path regardless. It's much more clear to act explicitly for reasons we know about. For example:
```
if [ "$SPREAD_SYSTEM" = "ubuntu-14.04-64" ]; then 
    ...
else
    ...
fi
```